galleries/examples/images_contours_and_fields/colormap_normalizations.py

```diff
@@ -13,85 +13,90 @@
+                   norm=colors.LogNorm(vmin=Z.min(), vmax=Z.max()))
+fig.colorbar(pcm, ax=ax[1], extend='max', label='LogNorm')
 
 # %%
-# PowerNorm: Here a power-law trend in X partially obscures a rectified
-# sine wave in Y. We can remove the power law using a PowerNorm.
+# PowerNorm
+# ---------
+# This example data mixes a power-law trend in X with a rectified sine wave in Y. If
+# plotted using a linear colour scale, then the power-law trend in X partially obscures
+# the sine wave in Y.
+#
+# The power law can be removed using a `.PowerNorm`.
 
 X, Y = np.mgrid[0:3:complex(0, N), 0:2:complex(0, N)]
-Z1 = (1 + np.sin(Y * 10.)) * X**2
+Z = (1 + np.sin(Y * 10)) * X**2
 
 fig, ax = plt.subplots(2, 1)
 
-pcm = ax[0].pcolormesh(X, Y, Z1, norm=colors.PowerNorm(gamma=1. / 2.),
-                       cmap='PuBu_r', shading='nearest')
-fig.colorbar(pcm, ax=ax[0], extend='max')
+pcm = ax[0].pcolormesh(X, Y, Z, cmap='PuBu_r', shading='nearest')
+fig.colorbar(pcm, ax=ax[0], extend='max', label='linear scaling')
 
-pcm = ax[1].pcolormesh(X, Y, Z1, cmap='PuBu_r', shading='nearest')
-fig.colorbar(pcm, ax=ax[1], extend='max')
+pcm = ax[1].pcolormesh(X, Y, Z, cmap='PuBu_r', shading='nearest',
+                       norm=colors.PowerNorm(gamma=0.5))
+fig.colorbar(pcm, ax=ax[1], extend='max', label='PowerNorm')
 
 # %%
-# SymLogNorm: two humps, one negative and one positive, The positive
-# with 5-times the amplitude. Linearly, you cannot see detail in the
-# negative hump.  Here we logarithmically scale the positive and
-# negative data separately.
+# SymLogNorm
+# ----------
+# This example data has two humps, one negative and one positive, The positive hump has
+# 5 times the amplitude of the negative. If plotted with a linear colour scale, then
+# the detail in the negative hump is obscured.
+#
+# Here we logarithmically scale the positive and negative data separately with
+# `.SymLogNorm`.
 #
 # Note that colorbar labels do not come out looking very good.
 
 X, Y = np.mgrid[-3:3:complex(0, N), -2:2:complex(0, N)]
-Z = 5 * np.exp(-X**2 - Y**2)
+Z1 = np.exp(-X**2 - Y**2)
+Z2 = np.exp(-(X - 1)**2 - (Y - 1)**2)
+Z = (5 * Z1 - Z2) * 2
 
 fig, ax = plt.subplots(2, 1)
 
-pcm = ax[0].pcolormesh(X, Y, Z,
-                       norm=colors.SymLogNorm(linthresh=0.03, linscale=0.03,
-                                              vmin=-1.0, vmax=1.0, base=10),
-                       cmap='RdBu_r', shading='nearest')
-fig.colorbar(pcm, ax=ax[0], extend='both')
+pcm = ax[0].pcolormesh(X, Y, Z, cmap='RdBu_r', shading='nearest',
+                       vmin=-np.max(Z))
+fig.colorbar(pcm, ax=ax[0], extend='both', label='linear scaling')
 
-pcm = ax[1].pcolormesh(X, Y, Z, cmap='RdBu_r', vmin=-np.max(Z),
-                       shading='nearest')
-fig.colorbar(pcm, ax=ax[1], extend='both')
+pcm = ax[1].pcolormesh(X, Y, Z, cmap='RdBu_r', shading='nearest',
+                       norm=colors.SymLogNorm(linthresh=0.03, linscale=0.03,
```

Comment:
```suggestion
                       norm=colors.SymLogNorm(linthresh=0.015,
```
works around the overlapping colorbar ticks 😄 

![grafik](https://github.com/matplotlib/matplotlib/assets/2836374/d4f07528-9308-4b7f-a980-86c1607ac4a2)